moveit_core/robot_state/test/robot_state_benchmark.cpp

```diff
@@ -32,132 +32,342 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  *********************************************************************/
 
-/* Author: Robert Haschke */
+/* Author: Robert Haschke, Mario Prats */
+
+// To run this benchmark, 'cd' to the build/moveit_core/robot_state directory and directly run the binary.
```

Comment:
Consider clarifying what is benchmarked here

Reply:
Added a general comment here, and additional comments before each of the benchmarks.